app/constants/benefits-enum.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  'income-support': {'requireBenefitUpload': false, 'displayName': 'Income Support'},
+  'jobseekers-allowance': {'requireBenefitUpload': false, 'displayName': 'Jobseeker’s Allowance (Income based)'},
+  'employment-support': {'requireBenefitUpload': false, 'displayName': 'Employment and Support Allowance (Income related)'},
+  'universal-credit': {'requireBenefitUpload': false, 'displayName': 'Universal Credit (and you earn less than £1,250 a month)'},
+  'working-tax-credit': {'requireBenefitUpload': true, 'displayName': 'Working Tax credits with disability or Child tax credit (and you earn less than £17,474 a year)'},
```

Comment:
Should the displayName be different for the internal site, i.e. remove brackets?